### PR TITLE
Installdocs 1

### PR DESCRIPTION
**Pull request by lynnbendixsen**
Node installation documents to install the server.

### CODE REVIEW

File: docs/source/node-add-troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Troubleshooting - Adding or Upgrading Indy Nodes
+Many things can go wrong while adding or upgrading nodes on an existing Indy network and this guide will cover symptoms and issues encountered and some steps you might take to recover from those. The steps listed are likely just possible remedies to the listed issues. Feel free to add more remedies or issues if you don't see your's included here. As bugs are fixed, the issues noted below might not occur any more, or might have a different remedy.
```
Comment (swcurran) [inline docs/source/node-add-troubleshooting.md:2]: Scary.  Perhaps just “Things an go wrong…"

File: docs/source/node-add-troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## Adding a Node
+This section covers troubleshooting the addition of a node to a network.  This can occur either as part of an upgrade (e.g. the 20.04 upgrade) or as part of a new node being added to an existing network.
+
+### Symptom 1 - Node is unresponsive  
+- Cause #1 - Node is performing catchup. (Large Network)
```
Comment (swcurran) [inline docs/source/node-add-troubleshooting.md:8]: Given the discussion on the Indy Contributor’s call this week (2023.09.26), should the instructions not always say to have a “recent” copy of the Indy Node data folder for the network, so the “long catchup” is not an issue?

When upgrading, the advice can be to use the node’s own data folder.

When adding a new node, include a step where the node operator’s technical team provides a data folder from another node.

File: docs/source/install-docs/Physical-NodeInstall-20.04 [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+            2. Type in password1 for your password
+            3. On successful login, type in ‘passwd’ to change your password on the Validator Node. Please use a unique password of sufficient length and store it in a secure place (i.e. a password manager). 
+            4. To set up 2FA, type in ‘google-authenticator’
+                1. Answer ‘y’ to all questions asked during the setup
+                2. Save the secret key, verification code, and scratch codes in a safe place. These are all for your user and can be used to login or to recover as needed.
+            5. Install Google Authenticator, Duo, Authy, or other google-authenticator compatible app on your phone or device.
+            6. On your 2FA phone app, add an account, and then scan the barcode or enter the 16 character secret key from step 4’s output.
+            7. Log out and then log back in to check and make sure it worked!
+    5. All of your secondary admin users should be setup now.
+30. You can now begin the Indy Node installation using the Validator Preparation Guide.
```
Comment (swcurran) [inline docs/source/install-docs/Physical-NodeInstall-20.04:219]: Is there a link available to the guide?

**Comment by swcurran** [comment]
Awesome stuff.  I added a couple of notes — one of which would seem to be important — the use of a data folder for the network when adding or upgrading a network.

File: docs/source/install-docs/Physical-NodeInstall-20.04 (hunk 3 above)
Comment (lynnbendixsen) [reply docs/source/install-docs/Physical-NodeInstall-20.04:219]: not yet, a future PR will have the guide in it

File: docs/source/node-add-troubleshooting.md (hunk 2 above)
Comment (lynnbendixsen) [reply docs/source/node-add-troubleshooting.md:8]: Wade has not experienced the issue, it seems, so I wondered whether "every time" is appropriate for all networks.  Until we figure out why this is only happening for me, maybe it's okay to leave this as is?
"Own" data folder is a great idea for the upgrade scenario.

**Review by swcurran — APPROVED** [review]

**Comment by swcurran** [comment]
To clairfy my approval. I didn’t go through the steps in detail — just assume that a “best effort” was done in creating the steps, and from this basis we can evolve the steps further.

**Review by swcurran — APPROVED** [review]

**Pull request merged**